mpmath/libmp/libmpf.py

```diff
@@ -327,7 +327,7 @@ def from_man_exp(man, exp, prec=None, rnd=round_fast):
 if BACKEND == 'sage':
     from_man_exp = sage_utils.from_man_exp
 
-def from_int(n, prec=0, rnd=round_fast):
+def from_int(n, prec=None, rnd=round_fast):
```

Comment:
This breaks gmpy builds.
```suggestion
def from_int(n, prec=0, rnd=round_fast):
```